### DISCUSSION TITLE
Got this warning: 'Update NavigationAuthority bound path tried to update multiple times per frame.'

### DISCUSSION THREAD

**Discussion by iRILLLL** [pointfreeco/swift-composable-architecture#2514] [Q&A]
Hi, there.

I am trying to add `NavigationStackStore` onto `TabView`. But I got this warning whenever I run the app. I have no idea and got stuck trying to find what might be the issue.
```
Update NavigationAuthority bound path tried to update multiple times per frame.
```

Here's snippet of my code. 
```swift
WithViewStore(self.store, observe: \.selectedTab) { viewStore in
    NavigationStackStore(self.store.scope(state: \.path, action: AppFeature.Action.path)) {
        TabView(selection: viewStore.binding(send: AppFeature.Action.selectedTabChanged)) {
        // ...
        }
    }
}
```
You can see the full code [here](https://github.com/iRILLLL/Movie/blob/main/features/Sources/MovieApp/AppView.swift#L23).

Does anybody have similar experience? how can I fix the warning.

regards,

**Comment by mbrandonw** [comment]
Hi @iRILLLL, unfortunately this is also an issue with vanilla SwiftUI, and I'm not sure of a fix. It _might_ help to move the `WithViewStore` inside the `NavigationStackStore`, but that's only a guess.

**Comment by iRILLLL** [comment]
Hi @mbrandonw thanks for taking your time to replied.

After updating to Xcode 15, the warning disappears (at least for now).

regards,

**Reply by Alexminator99** [reply]
Is happening for me using the last TCA version and Xcode 15.2 with iOS 17. Only once in a while though.

**Reply by mbrandonw** [reply]
Hi @Alexminator99, can you please provide a sample project that reproduces the behavior? There isn't much anyone can do otherwise.

**Reply by Alexminator99** [reply]
I will try to make a small project and share it.
